tests/expected_report.txt

```diff
@@ -8,7 +8,7 @@ tests/input/concatenation_spacing.php                 24      0
 tests/input/EarlyReturn.php                           4       0
 tests/input/example-class.php                         22      0
 tests/input/forbidden-comments.php                    4       0
-tests/input/forbidden-functions.php                   6       0
+tests/input/forbidden-functions.php                   7       0
```

Comment:
This change looks unrelated, where does it come from?

Reply:
Maybe doesn't require separate file, but it doesn't make sense to have it in "forbidden-functions.php". "example-class.php" would be better place if you really want to avoid new file.